dev_scripts/install

```diff
@@ -35,7 +35,7 @@ EOF
 	if [ ! -z $XDG_RUNTIME_DIR ] && [ -r "$XDG_RUNTIME_DIR" ]
 	then
 		mkdir -p "$XDG_RUNTIME_DIR/munin-sandbox-var"
-		[ -e "$XDG_RUNTIME_DIR/munin-sandbox-var" ] || ln -sv "$XDG_RUNTIME_DIR/munin-sandbox-var" sandbox/var
+		[ -e "$XDG_RUNTIME_DIR/munin-sandbox-var" ] && ln -sv "$XDG_RUNTIME_DIR/munin-sandbox-var" sandbox/var
```

Comment:
shell nitpicking: if the last command in a block (e.g. `if ... fi`) is a compound command (especially `&&`), then the success of the block depends on this compound command. Thus when using exit-on-error, the `if` block would cause the script to exit for no obvious reason. (the compound command itself would not cause the exit-on-error - but the `if` block would do this)
Anyway: it is not important here - I just wanted to share a subtile problem of this structure.